src/pudl/analysis/plant_parts_eia.py

```diff
@@ -730,6 +737,7 @@ def _clean_plant_parts(self, plant_parts_eia):
             )
             .set_index("record_id_eia")
         )
+        return plant_parts_eia[~plant_parts_eia.index.duplicated(keep="first")]
```

Comment:
This is the little patch to fix the one duplicated record from the new `TrueGranLabeler`. Adding it to this PR.